community/contributing/docs/github-changes.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "Make a Change Using the GitHub UI"
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Open a pull request on GitHub
+
+1. On the page where you see the issue, select the pencil icon at the top right. You can also scroll to the bottom of the page and select **Edit this page**.
+1. Make your changes in the GitHub markdown editor.
+1. Below the editor, fill in the **Propose file change** form.
+
+    In the **first field**, give your commit message a title that explains what your pull request is about.
+
+    The Spinnaker repositories use a PR title checker, so the title of your PR must follow a specific format: `<type>(<scope>): <subject>`
```

Comment:
Missing a period after subject.

Reply:
Done! I've made this change and it's now a new commit.